weaviate/exceptions.py

```diff
@@ -146,3 +146,13 @@ def __init__(self, message: str):
         msg = f"""gRPC call failed with message {message}."""
         super().__init__(msg)
         self.message = message
+
+
+class WeaviateAddProperty(WeaviateBaseError):
```

Comment:
To be as explicit as possible, should we call this `WeaviateAddInvalidPropertyError`? (With the `Error` prefix here conforming to [PEP8](https://peps.python.org/pep-0008/#exception-names); do we also subscribe to PEP8?)

Reply:
renamed - we didn't really follow it up to now but we can do in the future